tests/src/zigevm.test.ts

```
@@ -0,0 +1,79 @@
+/**
+ * Basic tests for ZigEVM TypeScript bindings
+ */
+import { describe, it, expect, beforeAll, afterAll } from 'vitest';
+import path from 'path';
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)